docs/examples/notebooks/audio_decoder.ipynb

```diff
@@ -0,0 +1,199 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "30b7205e-8719-4b16-8839-7b52c435d0da",
+   "metadata": {},
+   "source": [
+    "This is a simple example show-casing how to set up a simple audio decoder pipeline.\n",
+    "\n",
+    "We load and decode audio data using rocAL.\n",
+    "\n",
+    "The input data we will be using to show-case this example is a speech dataset which is present in the form of wav files."
```

Comment:
remove show-case (there is no hyphen). Change to "input data used for this example is a sample speech dataset available in .. as .wav file)